.github/PULL_REQUEST_TEMPLATE/feature_template.md

```
@@ -0,0 +1,57 @@
+
+Please confirm that you have completed the following:
+
+<!-- Add a small letter x to the square brackets to tick the checkbox -->
+
+- [ ] I have tested the feature thoroughly.
+- [ ] I have provided a clear and concise description of the feature implementation.
+- [ ] I have followed the code style and formatting guidelines.
+- [ ] I have added any necessary documentation or comments.
+- [ ] I have updated any relevant tests or test cases.
+
+#### Check only those which are applicable.
```

Markdownlint (0.35.0)

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)